README.md

```diff
@@ -152,6 +110,46 @@ const plugin = createPlugin({
 
 * `[captureIntrospectionQueries = false]` Enable capture of timings for an [IntrospectionQuery](https://graphql.org/graphql-js/utilities/#introspectionquery).
 
+### Apollo Federation Support
+
+The New Relic plugin will work with an Apollo Federated Server out of the box. Just pass the plugin to the Federated Gateway Apollo Server. With the federated server you will get:
+* Distributed Tracing support
+* Transaction naming
+* Operation naming
+* Metrics
+
+Resolver spans are not supported for the Federated Gateway Server. We strongly recommend adding the plugin to your sub-graph servers which will generate all the data you get with the Federated Server as well as resolver spans.
+
+```js
+// Federated Gateway Server index.js
+const plugin = require('@newrelic/apollo-server-plugin')
+
+const gateway = new ApolloGateway({
+  serviceList: [
+    { name: 'server1', url: 'server1' },
+    { name: 'server2', url: 'server2' }
```

Comment:
Super minor / can not worry about but while we are in here we could make he URL's look slightly more real-world. Either via var replacement or a fake URL.

```suggestion
    { name: 'server1', url: SERVER_1_URL },
    { name: 'server2', url: SERVER_2_URL }
```